packages/mds-jurisdiction-service/migrations/1582294819607-CreateJurisdictionsTable.ts

```diff
@@ -21,13 +21,19 @@ export class CreateJurisdictionsTable1582294819607 implements MigrationInterface
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `CREATE TABLE "jurisdictions" ("id" bigint GENERATED ALWAYS AS IDENTITY, "recorded" bigint NOT NULL, "jurisdiction_id" uuid NOT NULL, "agency_key" character varying(63) NOT NULL, "versions" json NOT NULL, CONSTRAINT "jurisdictions_pkey" PRIMARY KEY ("jurisdiction_id"))`,
```

Comment:
Was this change generated by TypeORM, or did you add it manually? If you added it manually, we'll have to think of a way to automate this for any new TypeORM services I think.

Reply:
In this case it was added manually, but that is only because I decided on an approach of using a separate migration table per service. Since I wanted to rename the jurisdiction migration table to be consistent, I needed the migration to be able to re-run. Normally the migrations are generated although they occasionally need to be tweaked in minor ways.